openlibrary/templates/covers/saved.html

```diff
@@ -2,22 +2,12 @@
 
 $putctx("cssfile", "form")
 
-<script type="text/javascript">
-<!--
-window.q.push(function() {
-    \$("#topNotice").hide();
-    \$("#popClose").click(closePopup);
-    parent.update_image("$(image and image.id)");
-});
-//-->
-</script>
-
 <div class="imagePost">
     $if image:
         <img src="$image.url(size='M')" class="cover" alt="New book cover"/>
 </div>
 
-<div class="imageSaved">
+<div class="imageSaved", data-config="$(image and image.id)">
```

Comment:
maybe name this data-image-id 
config suggests an array/object and I believe this is a string